.github/workflows/tests.yml

```
@@ -0,0 +1,34 @@
+  cancel-in-progress: true
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    strategy:
+      matrix:
+        python-version: ["3.12"]
+
+    steps:
+    - uses: actions/checkout@v4
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 6 but found 4

(indentation)